tests/scout_apm_tests/api/rename_transaction_test.py

```diff
@@ -0,0 +1,12 @@
+import scout_apm.api
```

Comment:
I think the test should be in `integration/test_api.py` with all the other tests for the public intefrace